pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateFieldRule.java

```diff
@@ -121,4 +136,28 @@ private boolean actuallyUsed(List<NameOccurrence> usages) {
     private boolean isOK(String image) {
         return "serialVersionUID".equals(image) || "serialPersistentFields".equals(image) || "IDENT".equals(image);
     }
+
+    /**
+     * Checks whether the given node is annotated with annotation in the set.
+     * The node should be field declaration.
+     *
+     * @param node
+     *            the node to check
+     * @return <code>true</code> if the annotation has been found
+     */
+    protected boolean hasNeglectAnnotation(Node node) {
+        boolean result = false;
+        Node parent = node.jjtGetParent();
+        List<ASTAnnotation> annotations = parent.findChildrenOfType(ASTAnnotation.class);
+        for (ASTAnnotation annotation : annotations) {
+            ASTName name = annotation.getFirstDescendantOfType(ASTName.class);
+            if (name != null) {
+                String annotationName = name.getImage();
+                if (NEGLECT_ANNOTATIONS.contains(annotationName)) {
+                    result = true;
```

Comment:
A more idiomatic way to write that would be to `return true` here, and `return false` as the last statement of the method, thus removing the `result` variable and ending the loop as soon as we know the method will succeed

Reply:
Done.